js_modules/dagit/packages/core/src/app/AppError.tsx

```diff
@@ -12,34 +12,61 @@ interface DagsterGraphQLError extends GraphQLError {
 
 const ErrorToaster = Toaster.create({position: 'top-right'});
 
-const showGraphQLError = (error: DagsterGraphQLError, operationName?: string) => {
+const showGraphQLError = (
+  error: DagsterGraphQLError,
+  operationName?: string,
+  networkPathPrefix?: string,
+) => {
   const message = (
     <div>
       Unexpected GraphQL error
-      <AppStackTraceLink error={error} operationName={operationName} />
+      <AppStackTraceLink
```

Comment:
Could we rename this component? It's kinda weird that it's called "Link" when it's not actually an Apollo client "link". Maybe just `AppStackTrace`